tests/controls/gridtest.cpp

```diff
@@ -742,4 +744,58 @@ void GridTestCase::ReadOnly()
 #endif
 }
 
+void GridTestCase::WindowAsEditorControl()
+{
+#if wxUSE_UIACTIONSIMULATOR
+    //The simple editor using wxWindow as a control
+    class TestEditor : public wxGridCellEditor
+    {
+    public:
+        TestEditor() = default;
```

Comment:
This won't compile with pre-C++11 compilers.

Also `override` below must be replaced with `wxOVERRIDE`.